.nuspec/Xamarin.Forms.targets

```diff
@@ -5,8 +5,12 @@
 
 	<PropertyGroup>
 		<EnableDefaultXamlItems Condition="'$(EnableDefaultXamlItems)'==''">True</EnableDefaultXamlItems>
+		<_DefaultXamlItemsEnabled>False</_DefaultXamlItemsEnabled>
+		<_DefaultXamlItemsEnabled Condition="'$(EnableDefaultItems)'=='' And '$(EnableDefaultXamlItems)'=='True' And '$(EnableDefaultEmbeddedResourceItems)'=='True'">True</_DefaultXamlItemsEnabled>
 	</PropertyGroup>
 
+	<Import Project="$(MSBuildThisFileDirectory)Xamarin.Forms.DefaultItems.targets" Condition="'$(_DefaultXamlItemsEnabled)'=='True" />
```

Comment:
Condition="'$(_DefaultXamlItemsEnabled)'=='True" should be Condition="$(_DefaultXamlItemsEnabled)'=='True"